src/components/home/sections/volunteer.scss

```diff
@@ -1,20 +1,26 @@
 $dark-purple: #412F8A;
 
-#volunteer {
+#volunteer { 
+  width: 100%; 
   @media only screen and (min-width: 1024px) {
     flex-direction: row;
-
+    float: left; 
+    padding: 1em;
     @media only screen and (min-width: 1366px) {
-      width: 80%;
-      margin: auto;
+      width: 40%;
+      margin: auto; 
+      padding-left: 20vh;
     }
   }
 
   .volunteer-content {
```

Comment:
Duplicate class name from boardmember.scss

Reply:
I am thinking this was due to the nesting nature of scss. I believe the class name will have different attributes if it has a different parent so to speak.

Reply:
Even if they are in different components and React will automate some things once everything has been processed, it's still good practice to be clearer and more intentional about avoiding collisions w/ naming.